### PR TITLE
Create metal.toml

### PR DESCRIPTION
**Pull request by lunchboxav**
Added a new library, filled with radio stream in genre of metal. Tested and it works fine, but please let me know if it doesn't work.

### CODE REVIEW

File: metal.toml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+name = "Bulldogs-Radio"
+url = "http://us4.internet-radio.com:8062/listen.pls"
+
+[idobi-howl]
+name = "idobi Howl"
+url = "http://69.46.88.26:80/listen.pls"
+
+[metal-express-radio]
+name = "Metal Express Radio"
+url = "http://5.135.154.69:11590/listen.pls
```
Comment (yuvadm) [inline metal.toml]: Missing `"`

File: metal.toml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+name = "Hand of Doom Radio"
+url = "http://158.69.114.190:8042/listen.pls"
+
+[metal-devastation-radio]
+name = "Metal Devastation Radio"
+url = "http://188.40.32.140:8061/listen.pls"
+
+[decennial-gothica-radio]
+name = "Decennial Gothica Radio
+url = "http://23.29.71.154:8024/listen.pls"
```
Comment (yuvadm) [inline metal.toml:67]: Missing `"`

**Review by yuvadm — CHANGES_REQUESTED** [review]
Please fix the two syntax errors

I should probably set up a Github Action to do this automagically

**Comment by lunchboxav** [comment]
Oops sorry, I think I messed that file up before committing. Fixed that.

**Pull request merged**